index.d.ts

```diff
@@ -1,6 +1,22 @@
 import {GlobbyOptions} from 'globby';
 
 declare namespace del {
+	interface ProgressData {
+		/**
+		Deleted file count.
+		*/
+		deletedFiles: number;
```

Comment:
I don't remember, but I think `del` also deal with folders (please check), which means `files` is technically incorrect.

Reply:
> I think del also deal with folders (please check)

Yes `del` also can deal with `directory`.

> which means files is technically incorrect.

It seems I need to think about renaming the properties.

I think `deletedFilesAndDirectories` could be too verbose.

How about `deletedCount` or `deleteCount` instead of `deletedFiles`?

And I think maybe `total` might be enough instead of `totalFiles`.

Reply:
`deletedCount` and `totalCount`?